engine/agg_tagset_cursor.go

```diff
@@ -875,7 +875,12 @@ func (s *AggTagSetCursor) SinkPlan(plan hybridqp.QueryNode) {
 			}
 		}
 		s.aggOps = callOps
-		s.baseCursorInfo.keyCursor.SinkPlan(plan.Children()[0].Children()[0])
+		_, ok := plan.Children()[0].Children()[0].(*executor.LogicalAggregate)
```

Comment:
Beware of out-of-bounds array.

Reply:
There will be no out-of-bounds situation, because plan.Children()[0] was of type agg before, and type agg must have children.